packages/core/test/stylable-transformer/general.spec.ts

```diff
@@ -16,124 +16,6 @@ describe('Stylable postcss transform (General)', () => {
         expect(result.toString()).to.equal('');
     });
 
-    it('should hoist :imports', () => {
```

Comment:
maybe we want to add these hoist tests as regression tests.

Reply:
simplified alternative hoist tests are in the `st-import.spec` and each feature that uses imports should define a simple test to check import interaction (hoist, override. redeclare diagnostic, etc...)